Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -33,11 +51,12 @@ class BaseFeatureWriter:
     tableTag = None
     features = frozenset()
     mode = "skip"
+    insertFeatureMarker = INSERT_FEATURE_MARKER
     options = {}
 
     _SUPPORTED_MODES = frozenset(["skip", "append"])
 
-    def __init__(self, features=None, mode=None, **kwargs):
+    def __init__(self, features=None, mode=None, insertFeatureMarker=None, **kwargs):
```

Comment:
unclear to me how useful would be to allow user to override the default insertion marker comment

Reply:
I removed the insertFeatureMarker argument for now.